lib/manager/setup-cfg/extract.spec.ts

```diff
@@ -10,24 +10,31 @@ describe('manager/setup-cfg/extract', () => {
       const res = extractPackageFile(Fixtures.get('setup-cfg-1.txt'));
       expect(res).toMatchSnapshot({
         deps: [
-          { depName: 'qux', currentValue: '>=4.4.4' },
```

Comment:
Why this is removed? This looks like a valid dependency to me.

Reply:
I think, it is a good idea to remove all misleading dependencies in
[`setup-cfg-1.txt`](https://github.com/renovatebot/renovate/blob/08acc9ad0e14c631b5674a833c5ad2ba6b36705f/lib/manager/setup-cfg/__fixtures__/setup-cfg-1.txt)
e.g.:
```
foo >=1.1.1
bar = bar >=2.2.2
[options]
baz >=3.3.3.
qux = qux >=4.4.4
quux >=5.5.5
```
This depedencies are all wrong.

Dependecies are in options must be after `install_require` or `[options.extras_require]`. Please check below.
```
[options]
install_require = 
  foo >=1.1.1
  baz >=3.3.3
  quux >=5.5.5
[options.extras_require]
  bar = bar >=2.2.2
  qux = qux >=4.4.4
```